install.yaml

```
@@ -8,8 +8,8 @@ pre_install_actions:
     #ddev-description:Check architecture type for incompatible arm64 type
     if [ "$(uname -m)" = "arm64" -o "$(uname -m)" = "aarch64" ]; then
       echo "This package does not work on arm64 machines"
-      if [ "${DDEV_IGNORE_ARCH_CHECK:-false}" != "true" ]; then
-        exit 1
+      if [ "${DDEV_IGNORE_ARCH_CHECK:-false}" != "true" ] && [ "$(arch)" = "arm64" -o "$(arch)" = "aarch64" ]; then
```

Revert it because this condition is already specified two lines above:

```suggestion
      if [ "${DDEV_IGNORE_ARCH_CHECK:-false}" != "true" ]; then
```

---

But we don't want to show the warning on amd64...

---

Oh thanks. It was just bad editing anyway, turned into simpler stanza.